ledger-api/grpc-definitions/com/daml/ledger/api/v1/command_completion_service.proto

```diff
@@ -64,6 +64,8 @@ message CompletionStreamRequest {
   string application_id = 2;
 
   // Non-empty list of parties whose data should be included.
+  // Only completions of commands for which the set of ``act_as`` parties is a subset of the given parties
```

Comment:
That's not correct though, is it? The stream contains completions of transactions for which the intersection between `parties` and `transaction.act_as` is non-empty. Otherwise this would break existing applications.

Reply:
Good point, this comment originates from the time where I thought it should be a subset check and needs to be changed.

> Otherwise this would break existing applications.

For single-party submissions, "act_as is a subset of completion request parties" and "act_as intersects completion request parties" is identical, no?

Reply:
Yes, you are right. Thanks.